packages/ra-core/src/dataProvider/useGetMany.spec.tsx

```diff
@@ -325,14 +349,6 @@ describe('useGetMany', () => {
             })
         );
         expect(hookValue.mock.calls[3][0]).toStrictEqual(
-            expect.objectContaining({
-                data: undefined,
-                isError: false,
-                isFetching: true,
-                isLoading: true,
-            })
-        );
-        expect(hookValue.mock.calls[4][0]).toStrictEqual(
```

Comment:
Why do we have one less state?

Reply:
Misteries of the new react-query or react